server/views/pages/standardLicenceConditions.njk

```diff
@@ -0,0 +1,12 @@
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+{%  macro standardLicenceConditions() %}
```

Comment:
can we call this immediate release conditions? 
Just that standard licence conditions sounds like the standard conditions you get on any licence